integration/admin/admin_integration_test.go

```diff
@@ -36,6 +36,7 @@ import (
 	"google.golang.org/grpc/status"
 
 	sa "github.com/google/trillian/server/admin"
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
```

Comment:
if you're going to alias this import might it as well become something shorter?

Reply:
The import is aliased to the name of the go package that `go-grpc-middleware` declares. 
I could rename it but `grpc_middleware` seems pretty clear and standard.